extension/js/common/platform/xss.ts

```diff
@@ -64,8 +68,8 @@ export class Xss {
     return cleanHtml;
   }
 
-  public static htmlSanitizeAndStripAllTags = (dirtyHtml: string, outputNl: string): string => {
-    let html = Xss.htmlSanitizeKeepBasicTags(dirtyHtml);
+  public static htmlSanitizeAndStripAllTags = (dirtyHtml: string, outputNl: string, removeImgs: boolean = false): string => {
```

Comment:
Can you please share more info about how `removeImgs` is used, when and why? This would probably make for a good comment